wren-ai-service/tests/pytest/services/mocks.py

```diff
@@ -1,7 +1,7 @@
 from typing import Optional
 
 from src.pipelines.generation import intent_classification, sql_generation, sql_summary
-from src.pipelines.retrieval import historical_question, retrieval
+from src.pipelines.retrieval import historical_question_retrieval, retrieval
```

Comment:
i think we could directly access from generation and retrieval module. could you also modify the import?

Reply:
fixed